releases/v0.16.0.md

```diff
@@ -0,0 +1,92 @@
+# Release Notes for v0.16.0
+
+Release Date: September 25th, 2025
+
+This release is based on [3.21.0](https://github.com/kata-containers/kata-containers/releases/tag/3.21.0) of Kata Containers
+and [v0.11.0](https://github.com/confidential-containers/enclave-cc/releases/tag/v0.11.0) of enclave-cc.
+
+Trustee and the guest components use KBS protocol v0.4.0.
+
+Please see the [quickstart guide](https://confidentialcontainers.org/docs/getting-started/) or [project documentation](https://confidentialcontainers.org/docs) for more information.
+
+## Deprecation Notices
+* Support for process-based confidential computing via enclave-cc will be removed in the v0.18.0 release.
+  The enclave-cc project will be archived.
+* This will be the last release of CoCo and Trustee that supports `simple` attestation tokens.
+  Today, EAR attestation tokens are the default, but Trustee can be configured to use `simple` tokens.
+  This option will be removed.
+
+## What's New
+
+* The affirming resource policy now checks that every submod is affirming. With multi-device attestation,
+  KBS policies should be aware of all submods.
+* Experimental support for attesting some NVIDIA GPUs, such as the H100. Either ITA or the Trustee
+  attestation service can be used to verify the device evidence, with some limitations.
+* Trustee supports AMD-SNP guests with report version 5.
+* Additional/improved TCB claims generated by TDX attester.
+* Extractor modules can now receive configuration if required.
+* SWID/RIM-IM extractor improved.
+* CSV verifier supports AAEL.
+* Eventlog parsing supports SM3 hashes.
```

Comment:
we also dropped the AAEL as a custom eventlog and merged it with CC eventlog. attestation-agent measured events are now TCG2 `EV_EVENT_TAG`  events using tag ID `0x4141454c` ("AAEL"). AAEL events are added as EAR token claims after Trustee has verified the event description matches the measurement hash.